streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/impl/KubernetesSessionClientV2.scala

```diff
@@ -133,4 +165,115 @@ object KubernetesSessionClientV2 extends KubernetesClientV2Trait with Logger {
     }
   }
 
+  private def genFlinkConfig(deployRequest: DeployRequest): Configuration = {
```

Comment:
The `genFlinkConfig` function is not necessary, just only keep `extractConfiguration(deployRequest.flinkVersion.flinkHome, deployRequest.properties)` as the initial value of Flink Configuration. 
Set these valid configurations directly in `genFlinkDeployDef` function. For the missing configurations in `FlinkDeployDef`, set them in `extraFlinkConfiguration`.

Reply:
At present, the extra FlinkConfiguration function will continue to be optimized in the future.